python/modules/Stop0lBaselineProducer.py

```diff
@@ -136,21 +138,29 @@ def PassNjets(self, jets):
 
     def GetJetSortedIdx(self, jets):
         ptlist = []
+	etalist = []
         dphiMET = []
         for j in jets:
             if math.fabs(j.eta) > 4.7 or j.pt < 20:
                 pass
             else:
                 ptlist.append(j.pt)
```

Comment:
```suggestion
                ptlist.append(-j.pt)
```

Reply:
We do not want to do `ptlist[::-1]` because reversing `ptlist` makes it not match `etalist` any more.

Here's what happens to the original example if we move the 50 GeV jet to somewhere other than the end:
```
ptlist =  array([ 10,  20,  20,  30,  50,  30])
etalist = array([0.1, 0.4, 0.2, 0.2, 0.7, 0.3])
sort = np.lexsort((abs(etalist), ptlist[::-1]))
print('"Sorted" indices:')
print(sort)
print('pt in "sorted" order:')
print(ptlist[sort])
print('eta in "sorted" order:')
print(etalist[sort])
```
output:
```
"Sorted" indices:
[5 3 4 0 2 1]
pt in "sorted" order:
[30 30 50 10 20 20]
eta in "sorted" order:
[0.3 0.2 0.7 0.1 0.2 0.4]
```

If we instead sort `-ptlist`, then it comes out ok:
```
ptlist =  array([ 10,  20,  20,  30,  50,  30])
etalist = array([0.1, 0.4, 0.2, 0.2, 0.7, 0.3])
sort = np.lexsort((abs(etalist), -ptlist))
print('"Sorted" indices:')
print(sort)
print('pt in "sorted" order:')
print(ptlist[sort])
print('eta in "sorted" order:')
print(etalist[sort])
```
output:
```
"Sorted" indices:
[4 3 5 2 1 0]
pt in "sorted" order:
[50 30 30 20 20 10]
eta in "sorted" order:
[0.7 0.2 0.3 0.2 0.4 0.1]
```

You can see that the latter version works properly.